content/en/vizs-and-tips/how-to-create-multiple-lags-in-R/index.html

```diff
@@ -0,0 +1,144 @@
+---
+title: 'Unlocking the Power of purrr: How to Create Multiple Lags Like a Pro in R'
+author: Pablo Cánovas Tourné
+date: '2023-03-22'
+slug: how-to-create-multiple-lags-in-R
+output: 
+  blogdown::html_page:
+    highlight: tango 
+categories:
+  - R
+  - Tips
+tags: [Tidyverse]
+subtitle: 'A quick guide to creating multiple lags in R for Time Series Analysis with purrr'
+summary: 'While R provides functions like lag, creating multiple lags can be challenging. In this blog post, we will explore how to use the unknown partial function to create multiple lags in R for Time Series Analysis'
+authors: [pablo-canovas]
+featured: false
+---
+

```

Comment:
faltaría aquí esto, ¿no?

```suggestion

<script src="{{< blogdown/postref >}}index_files/header-attrs/header-attrs.js"></script>
```